### ISSUE TITLE
Any plans for 1.03?

### ISSUE THREAD

**Issue by javaskript** [AFirstTimeMapMaker/ObjToCODMap#1]
Hi, I know you’ve been inactive for some time here on GitHub as well as ModMe, but are there any plans for a v1.03? Thanks

**Comment by warkingous** [comment]
> Hi, I know you’ve been inactive for some time here on GitHub as well as ModMe, but are there any plans for a v1.03? Thanks

Hello, did u manage to get this tool working? i would appreciate some help if u dont mind

**Comment by javaskript** [comment]
> Hello, did u manage to get this tool working? i would appreciate some help if u dont mind

Yes, the converter worked but was not ideal. I ended up moving over to [HammUEr](https://nte.itch.io/hammuer), converting Source engine maps into UE4, then converting that into a CoD map.
